### PR TITLE
VLANs on SR Linux part2

### PR DESCRIPTION
**Pull request by michelredondo**
Ready to be reviewed

### CODE REVIEW

File: docs/blog/posts/2024/srl-vlans2.md [hunk 1]
```diff
@@ -0,0 +1,765 @@
...
+<div class="mxgraph" style="max-width:100%;border:1px solid transparent;margin:0 auto; display:block;" data-mxgraph='{"page":2,"zoom":2,"highlight":"#0000ff","nav":false,"check-visible-state":true,"resize":true,"url":"https://raw.githubusercontent.com/srl-labs/srlinux-eos-vlan-handling-lab/diagrams/srl-eos-vlan.drawio"}'></div>
+
+### Scenario 3: Single-tagged-range VLAN
+
+Whenever we want the subinterface to accept frames with a range of VLAN IDs, we can use single-tagged-range encapsulation type.
+
+This mode doesn't pop any VLAN tags on ingress, nor it adds them on ingress. Subinterface will just filter the frames that have VLAN ID within the configured range and pass them through without any modifications.
+
+In this case, for the SR Linux `ethernet-1/10` interface, we have configured the subinterface with vlan-id `any`. So we can also check that tags won't be modified in any way.
+
```
Comment (hellt) [inline docs/blog/posts/2024/srl-vlans2.md:429]: @michelredondo why was this required? We originally had `vlan-tagging false` for inter switch links, is there a point in configuring this to `any`?

Comment (michelredondo) [reply docs/blog/posts/2024/srl-vlans2.md:429]: It's just to show an alternate configuration that provides vlan tagging at interface level and frame transparency.
Maybe we can add something like:
"in the previous post we used `vlan-tagging false' for inter switch links. In this scenario we are using vlan-id `any`, which also provides frame transparency, and it also allows us to use vlan tagging for a specific VLAN ID, at other subinterface in the same port".

Comment (hellt) [reply docs/blog/posts/2024/srl-vlans2.md]: understood, thanks
added this in [e2600e5](https://github.com/srl-labs/learn-srlinux/pull/146/commits/e2600e50a2d0667cc49c08bca0c9ac8048008a36)
